CONTRIBUTING.rst

```diff
@@ -65,7 +65,7 @@ First time setup
 - Create a virtualenv::
 
         python3 -m venv env
-        . venv/bin/activate
+        . env/bin/activate
         # or "venv\Scripts\activate" on Windows
```

Comment:
It's wrong here as well. What about just replacing `env` with `venv` when creating it (or `.venv`? that might be more common)

Reply:
I'm sorry. I think you're right.
I'll fix as below. 
old
```
python3 -m venv env
```
new
```
python3 -m venv venv
```